README.md

```diff
@@ -12,32 +12,116 @@
+
+```sh
+$ architect dev .
+```
+
+When this command completes, you can reach your new application at https://app.localhost.architect.sh.
+### Make your own changes
+
+This application's `architect.yml` file contains a `debug` block that enables hot-reloading for each service
+within the component. That means you can make changes to the source code, which will automatically apply to the environment. This lets you quickly iterate and see your changes without restarting the
+application stack.
+
+Give it a try! Search inside your project for "Favorite Movie” and change this string to “Favorite Pizza.” Once you saved
+the file, you’ll see the frontend service recompiling in the logs, and then your browser window will update automatically.
 
-# Deploy locally using the dev command
-$ architect dev architect.yml
+## Deploy to the Cloud
+Do you want to try deploying this application to a cloud environment? Architect's got you covered there, too!
+We offer free preview environments in our community cloud where you can deploy your applications
+before deploying to staging or prod. This is an excellent opportunity for testing and getting early feedback before merging
+your code. You can [configure your GitOps](https://docs.architect.io/tutorial/creating-a-component)
+to automatically deploy every PR to Architect's community cloud.
+
+### Create an environment
+
+To create a new environment on Architect's
+free cloud, run the following command:
+
+```sh
+architect environments:create my-first-environment
 ```
+This command presents you with a list of Kubernetes clusters. Since you haven't added external clusters to your
+account, you should only see `architect`. Hit enter to create your environment on Architect's community cloud.
 
-## Deploying to the Cloud
+```sh
+? Select a cluster (Use arrow keys or type to search)
+❯  architect
+```
+When the command completes, you should see output similar to the following:
+```sh
+%architect environments:create my-first-environment
+? Select a cluster architect
+Registering environment with Architect... done
+Environment created: https://cloud.architect.io/<account-name>/environments/my-first-environment
+```
 
-Want to try deploying this to a cloud environment? Architect's got you covered there, too! It only takes a minute to
-[sign up for a free account](https://cloud.architect.io/signup).
+### Deploy your component
 
-You can then [deploy the application](https://docs.architect.io/getting-started/introduction/#deploy-to-the-cloud) by running the command below. Note that “example-environment” is the free environment that is created with your Architect account.
+You are now ready to deploy your component to your environment in Architect's community cloud. To deploy your component,
+run the following command from the `nestjs` directory:
 
 ```sh
-# Deploy to Architect Cloud
-$ architect deploy architect.yml -e example-environment
+architect deploy --account <account-name> \
```

Comment:
Like in comments on other projects, this should either deploy `architect.yml` or include a line to register the component

Reply:
that's what this is doing now - `architect deploy --account <account-name> --environment my-first-environment ./architect.yml` should be the correct command, I can verify that it actually works because I can't remember for this project if i tried it or deployed manually lol

Reply:
Oh sorry, I always put that right after the command so I didn't notice it at the end